### PR TITLE
Ballot2/77

### PR DESCRIPTION
**Pull request by ryma2fhir**
Decision to revert to base standards for languages and separate Makaton into new ValueSet
- Added slice to language.coding
- Set language to all-languages base valueset as per R5 and bound it as required for forward compatibility (R4 currently set to a selection of languages - binding preferred but limited to all-languages)
- Changed UKCore Human Language valuset to include Makaton only from SCT. Retired CodeSystem
- Changed example to show Makaton in use

### CODE REVIEW

**Comment by APageNHS** [comment]
Can you look at Practitioner and RelatedPerson at the same time? Asa I think both of these either have a communication/language or are bound to humanlanguage

**Comment by ryma2fhir** [comment]
Will do. Unfortunately this is not validating
>Received: "ERROR Inappropriate CodeSystem URL "[http://snomed.info/sct"](https://simplifier.net/resolve?fhirVersion=R4&scope=uk.nhsdigital.r4&canonical=http://snomed.info/sct%22) for ValueSet: [http://hl7.org/fhir/ValueSet/all-languages](https://simplifier.net/resolve?fhirVersion=R4&scope=uk.nhsdigital.r4&canonical=http://hl7.org/fhir/ValueSet/all-languages) [ Location - Patient.communication[0].language] [ Location - Line 1, Col 3219]"

File: structuredefinitions/Extension-UKCore-NHSCommunication.xml [hunk 1]
```diff
@@ -79,9 +73,9 @@
         <code value="CodeableConcept" />
       </type>
       <binding>
-        <strength value="extensible" />
+        <strength value="required" />
         <description value="Human Language" />
-        <valueSet value="https://fhir.hl7.org.uk/ValueSet/UKCore-HumanLanguage" />
+        <valueSet value="http://hl7.org/fhir/ValueSet/all-languages" />
```
Comment (APageNHS) [inline structuredefinitions/Extension-UKCore-NHSCommunication.xml:78]: Do we need to change this extension, given its retired?

File: valuesets/ValueSet-UKCore-HumanLanguage.xml [hunk 2]
```diff
@@ -20,979 +20,31 @@
       <rank value="1" />
     </telecom>
   </contact>
-  <description value="A set of codes that define a language used by a person." />
+  <description value="A set of codes that define a language used by a person.Selected from the SNOMED CT UK coding system:
```
Comment (APageNHS) [inline valuesets/ValueSet-UKCore-HumanLanguage.xml:23]: Do we want to revert this valueset to its original contents if we are retiring it?

**Review by APageNHS — DISMISSED** [review]

File: structuredefinitions/UKCore-Patient.xml [hunk 3]
```diff
@@ -242,10 +242,6 @@
       </type>
       <mustSupport value="false" />
     </element>
-    <element id="Patient.address.extension:addressKey.extension">
-      <path value="Patient.address.extension.extension" />
-      <min value="2" />
-    </element>
```
Comment (ryma2fhir) [inline structuredefinitions/UKCore-Patient.xml:248]: Removed by forge

**Review by APageNHS — APPROVED** [review]

**Pull request merged**